File: documentation/user/en/query/ordering/examples/reference/entity-property.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeEquals("code", "garmin-vivoactive-4"),
					entityLocaleEquals(Locale.forLanguageTag("en"))
				),
				require(
					entityFetch(
						attributeContent("code"),
						referenceContent(
							"parameterValues",
							orderBy(
								entityProperty(
									attributeNatural("name", DESC)
								)
							),
							entityFetch(
								attributeContent("name")
							)
						)
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/ordering/examples/reference/entity-property.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeEquals("code", "garmin-vivoactive-4"),
        		EntityLocaleEquals(CultureInfo.GetCultureInfo("en"))
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code"),
        			ReferenceContent(
        				"parameterValues",
        				OrderBy(
        					EntityProperty(
        						AttributeNatural("name", Desc)
        					)
        				),
        				EntityFetch(
        					AttributeContent("name")
        				)
        			)
        		)
        	)
        )
	)
);